wrappers/python/wirepas_messaging/wnt/ws_api/messagesbase.py

```diff
@@ -62,6 +62,10 @@ class MessageTypes(enum.Enum):
         SEND_REMOTE_API_REQUEST = 1072
         SEND_DATA_MESSAGE = 1073
 
+    class ProtocolVersions(enum.Enum):
```

Comment:
What is the benefit in using a nested class instead of a class attribute? The enum will allow you to have keys with the same value (alias).

Reply:
Can you please elaborate a bit more. I have to say that I don't understand what you are saying now.

Reply:
Currently you have a class inside a class:

class MessageTypes():
      class ProtocolVersions():
           version_2=2
           __init__(self,args): # hidden

      __init__(self,args):

I was asking what is the benefit between your current approach and going with standard attributes:

Class MessageTypes():
      version_2=2 # class attribute
      __init__(self,args):

Reply:
They are under different naming then. We might have different enumerations and now they have distinct parent name when used. e.g. MessageTypes.ProtolVersions.VERSION_2

Reply:
And this is actually working properly from outside the class scope. 

Good, let's merge.